.mvn/maven.config

```diff
@@ -0,0 +1,4 @@
+-Daether.remoteRepositoryFilter.prefixes=true
```

Comment:
If we're introducing `maven.config`, I think we should also add these:
```
-T2C
--strict-checksums
```
Can you extract adding `maven.config` to a separate commit?

Reply:
Created https://github.com/trinodb/trino/pull/20479

Regarding `-T2C`, I would **not do that** without takari-smart-builder (used by mvnd as well), as out of the box parallel builder is inferior to it.

Reply:
Thanks! I found this page that explains the difference: http://takari.io/book/30-team-maven.html#takari-smart-builder. It sounds like we should try it out, but in a separate PR.

Reply:
Also the project is alive, and received patches, and as I mentioned, mvnd (Maven Daemon) is using it, so is active: https://github.com/takari/takari-smart-builder

Reply:
https://github.com/trinodb/trino/pull/20485

Reply:
BTW you can mark my comments as resolved, I can't do it myself.